build.gradle.kts

```diff
@@ -39,6 +43,7 @@ plugins {
     alias(libs.plugins.shadowJar)
     alias(libs.plugins.taskTree)
     alias(libs.plugins.hugo)
+    alias(libs.plugins.jpackage)
```

Comment:
Shouldn't we apply this solely to `alchemist-full`?

Reply:
Applying and configuring all the jpackage related stuff in the alchemist-full/build.gradle.kts?

Reply:
yes

Reply:
Shall I upload in the release all the JARs with the packages or only the packages? 
Packages provides an easy way to install and use alchemist-full. If the other modules' jar are useful we could upload the packages + all the other jars except the full one.

Reply:
At the moment I'd keep the current jars + the redistributable packages. It'd make sense, though, to ship only alchemist-full, as jar or platform-specific redistributable.